jenkins/common.groovy

```diff
@@ -156,19 +156,21 @@ def checkForBrokenLinks(String buildLogPath, String envName, Boolean contentOnly
   // Output a csv file with the broken links
   sh "cd /application && jenkins/glean-broken-links.sh ${buildLogPath} ${csvFileName}"
   if (fileExists(csvFile)) {
-    echo "Found broken links; notifying the Slack channel."
-    // TODO: Move this slackUploadFile to cacheDrupalContent and update the echo statement above
-    // slackUploadFile(filePath: csvFile, channel: 'dev_null', failOnError: true, initialComment: "Found broken links in the ${envName} build on `${env.BRANCH_NAME}`.")
-
-    // Until slackUploadFile works...
-    def linkCount = sh(returnStdout: true, script: "cd /application && wc -l ${csvFileName} | cut -d ' ' -f1") as Integer
-    slackSend message: "${linkCount} broken links found in the ${envName} build on `${env.BRANCH_NAME}`\n${env.RUN_DISPLAY_URL}".stripMargin(),
-      color: 'danger',
-      failOnError: true,
-      channel: 'cms-team'
+    echo "Found broken links."
 
     // Only break the build if broken links are found in master
     if (IS_PROD_BRANCH || contentOnlyBuild) {
+      echo "Notifying Slack channel."
```

Comment:
Nice, this seems like it should reduce some noise in the #cms-team channel!

Reply:
yup, hopefully now the only broken link alerts will be ones we actually care about :D